src/articles/dto/articles-response.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { Article } from '@prisma/client';
+import { Expose, Transform, TransformFnParams } from 'class-transformer';
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+function shortenText({ value: text }: TransformFnParams) {
+  if (!text) {
+    return null;
+  }
+  if (text.length > 100) {
+    return `${text.substring(0, 98)}${'.'.repeat(3)}`;
+  }
+  return text;
+}
+
+export function capitalizeTitle({ value }: TransformFnParams) {
```

Comment:
Since `capitalizeTitle` is used in another file as well, it should have a separate file called `capitalize-title.ts`